src/Symfony/Contracts/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+3.7
```

Comment:
I think we also need to bump the version in the root composer.json, maybe also in the composer.json of contracts - look for "3.6.x"